whelk-core/src/main/groovy/se/kb/libris/Normalizers.groovy

```diff
@@ -77,7 +77,11 @@ class Normalizers {
                         if (typeList.size() == 1)
                             node[key] = typeList[0]
                         else {
-                            throw new ModelValidationException("Could not reduce: " + typeList + " to a single type (required) by removing superclasses.")
+
+                            //throw new ModelValidationException("Could not reduce: " + typeList + " to a single type (required) by removing superclasses.")
+                            // This must be reduced to a warning, because the assumption does not hold true: There are records in
+                            // libris that legitimately have unrelated multi-types (see definitions).
+                            log.warn("Could not reduce: " + typeList + " to a single type by removing superclasses.")
```

Comment:
If that was only because of the `a owl:Class, rdfs:Datatype` items, we could do something about that in the definitions (also, `rdfs:Datatype rdfs:subClassOf rdfs:Class` but our system doesn't know that :/).

Reply:
I'm working on the other stuff, but for reference, here are the remaining multi types in definitions:

[Class, Datatype] to a single type by removing superclasses.
[Class, IssuanceType] to a single type by removing superclasses.
[FunctionalProperty, ObjectProperty] to a single type by removing superclasses.
[marc:CollectionClass, Class] to a single type by removing superclasses.
[marc:HoldingStatusType, marc:AuthorityStatusType] to a single type by removing superclasses.
[marc:I1codeSource, marc:LanguageCode, marc:I2sourceOfCode, marc:NumberSource] to a single type by removing superclasses.
[marc:SummaryType, marc:AudienceType] to a single type by removing superclasses.
[ObjectProperty, DatatypeProperty] to a single type by removing superclasses.
[ObjectProperty, owl:SymmetricProperty] to a single type by removing superclasses.